custom_components/nest_protect/manifest.json

```diff
@@ -1,18 +1,16 @@
 {
-    "domain": "nest_protect",
-    "name": "Nest Protect",
-    "config_flow": true,
-    "documentation": "https://github.com/imicknl/ha-nest-protect",
-    "issue_tracker": "https://github.com/imicknl/ha-nest-protect/issues",
-    "requirements": [],
-    "codeowners": [
-        "@imicknl"
-    ],
-    "iot_class": "cloud_polling",
-    "version": "0.3.8",
-    "dhcp": [
-        {
-            "macaddress": "CCA7C1*"
-        }
-    ]
-}
\ No newline at end of file
+  "domain": "nest_protect",
+  "name": "Nest Protect",
+  "config_flow": true,
+  "documentation": "https://github.com/imicknl/ha-nest-protect",
+  "issue_tracker": "https://github.com/imicknl/ha-nest-protect/issues",
+  "requirements": [],
+  "codeowners": ["@imicknl"],
+  "iot_class": "cloud_polling",
+  "version": "0.4.0",
```

Comment:
Please don't bump the version number by hand in your PR.

Reply:
This make shure it's not interfering with the nest integration of home assistant.

Reply:
@Cruguah I know, but in this integration we keep our pre-releases in main (latest git version) and new releases on GitHub will bump the version number.

Reply:
Removed the version number